test/unittest/reduction/reduction_test.cpp

```diff
@@ -151,12 +148,12 @@ void run_test(const combination_t<scalar_t> combi) {
             ex, m_in_gpu, ld, v_out_gpu, rows, cols);
         break;
       case operator_t::Max:
-        ev = extension::_reduction<MaxOperator, scalar_t>(ex, m_in_gpu, ld,
-                                                        v_out_gpu, rows, cols);
```

Comment:
Looks like this file was not formatted correctly. Can you double-check if the changes you added are in clang-format?